tests/functional/data/contracts/ethereum/solidity_contract_json.json

```
@@ -0,0 +1,205 @@
+    ,"outputs":[{"internalType":"uint256","name":"","type":"uint256"} 
+    ,{"components":[{"internalType":"uint256","name":"one" 
+    ,"type":"uint256"},{"internalType":"uint256","name":"two" 
+    ,"type":"uint256"},{"internalType":"uint256","name":"three" 
+    ,"type":"uint256"},{"internalType":"uint256","name":"four" 
+    ,"type":"uint256"},{"internalType":"uint256","name":"five" 
+    ,"type":"uint256"},{"internalType":"uint256","name":"six" 
+    ,"type":"uint256"}],"internalType":"struct TestContractSol.IntStruct[5]" 
+    ,"name":"","type":"tuple[5]"}],"stateMutability":"pure" 
+    ,"type":"function"},{"inputs":[],"name":"getUnnamedTuple" 
+    ,"outputs":[{"internalType":"uint256","name":"","type":"uint256"} 
+    ,{"internalType":"uint256","name":"","type":"uint256"}] 
+    ,"stateMutability":"pure","type":"function"},{"inputs":[] 
+    ,"name":"myNumber","outputs":[{"internalType":"uint256" 
+    ,"name":"","type":"uint256"}],"stateMutability":"view" 
+    ,"type":"function"},{"inputs":[],"name":"owner","outputs":[{"internalType":"address" 
+    ,"name":"","type":"address"}],"stateMutability":"view" 
+    ,"type":"function"},{"inputs":[],"name":"prevNumber" 
+    ,"outputs":[{"internalType":"uint256","name":"","type":"uint256"}] 
+    ,"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"address" 
+    ,"name":"_address","type":"address"}],"name":"setAddress" 
+    ,"outputs":[],"stateMutability":"nonpayable","type":"function"} 
+    ,{"inputs":[{"internalType":"address","name":"_address" 
+    ,"type":"address"},{"internalType":"uint256","name":"bal" 
+    ,"type":"uint256"}],"name":"setBalance","outputs":[] 
+    ,"stateMutability":"nonpayable","type":"function"} 
+    ,{"inputs":[{"internalType":"uint256","name":"num" 
+    ,"type":"uint256"}],"name":"setNumber","outputs":[] 
+    ,"stateMutability":"nonpayable","type":"function"} 
+    ,{"inputs":[{"internalType":"uint256","name":"num" 
+    ,"type":"uint256"},{"internalType":"address","name":"_address" 
+    ,"type":"address"}],"name":"setNumber","outputs":[] 
+    ,"stateMutability":"nonpayable","type":"function"} 
+    ,{"inputs":[{"components":[{"internalType":"address" 
+    ,"name":"a","type":"address"},{"internalType":"bytes32" 
+    ,"name":"b","type":"bytes32"}],"internalType":"struct TestContractSol.MyStruct","name":"_my_struct","type":"tuple"}] 
+    ,"name":"setStruct","outputs":[],"stateMutability":"pure" 
+    ,"type":"function"},{"inputs":[{"components":[{"internalType":"address" 
+    ,"name":"a","type":"address"},{"internalType":"bytes32" 
+    ,"name":"b","type":"bytes32"}],"internalType":"struct TestContractSol.MyStruct[2]","name":"_my_struct_array"
+    ,"type":"tuple[2]"}],"name":"setStructArray","outputs":[] 
+    ,"stateMutability":"pure","type":"function"},{"inputs":[] 
+    ,"name":"theAddress","outputs":[{"internalType":"address" 
+    ,"name":"","type":"address"}],"stateMutability":"view" 
+    ,"type":"function"}]
```

```suggestion
    ,"type":"function"}]

```

Missing newline at EOF!